### ISSUE TITLE
csg_orientcorr.rst:17: WARNING: Definition list ends without a blank line; unexpected unindent.

### ISSUE THREAD

**Issue by junghans** [votca/csg#639]
Warning when converting csg_orientcorr output:
```
csg_orientcorr.rst:17: WARNING: Definition list ends without a blank line; unexpected unindent.
```

The rst section:
```
    13	DESCRIPTION
    14	***********
    15	Calculates the orientational correlation function
    16	    <3/2*u(0)*u(r) - 1/2>
    17	for a polymer melt, where u is the vector pointing along a bond and
    18	r the distance between bond segments (centered on middle of bond).
    19
    20	OPTIONS
    21	*******
```

**Comment by felipeZ** [comment]
@junghans  We can solve this issue by moving the equation at the begginning of the line like:
```rst
Calculates the orientational correlation function
<3/2*u(0)*u(r) - 1/2>
for a polymer melt, where u is the vector pointing along a bond and
```

**Comment by junghans** [comment]
I thought we fixed that in #648
